app/services/search_rummager_service.rb

```diff
@@ -2,11 +2,9 @@ class SearchRummagerService
   def fetch_related_documents(filter_params = {})
     options = default_search_options.merge(filter_params)
 
-    Rails.cache.fetch(options, expires_in: 5.minutes) do
-      search_response = Whitehall.search_client.search(options)
-      search_response["results"].map! { |res| RummagerDocumentPresenter.new(res) }
-      search_response
-    end
```

Comment:
So I removed caching because it caused this error: `TypeError: no _dump_data is defined for class Thread::Mutex`.

I suspect this is because we're trying to serialise a mutex.

From the [Ruby docs](https://ruby-doc.org/core-2.7.0/Marshal.html): 

```
Some objects cannot be dumped: if the objects to be dumped include bindings, procedure or method objects, instances of class IO, or singleton objects, a TypeError will be raised.
```

I am investigating how to deal with this. Any suggestions warmly welcomed :)